lib/angular-evaporate.js

```diff
@@ -301,7 +302,9 @@
                * @returns - same as Evaporate.add()
                */
               AngularEvaporate.prototype.$start = function (upload) {
-
+                var args = Array.prototype.slice.call(arguments);
+                args[1] = upload.overrideConfig;
```

Comment:
And `args` being the shallow copy of `arguments` is not going to be used later, what's the point? Does this work for you?